net-misc/ncman/ncman-9999.ebuild

```diff
@@ -14,18 +17,16 @@ else
 	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
 fi
 
-DESCRIPTION="An ncurses UI for connman, forked from connman-json-client"
-HOMEPAGE="https://github.com/l4rzy/ncman"
-
 LICENSE="GPL-2"
```

Comment:
same here, should be `GPL-2+`, see: https://github.com/l4rzy/ncman/blob/master/src/main.c#L7-L9